README.md

```diff
@@ -2,11 +2,17 @@
 
 TensorSets are a [third party resource](http://kubernetes.io/docs/user-guide/thirdpartyresources/) to manage [Tensorflow](https://github.com/tensorflow) training clusters running in [Kubernetes](https://github.com/tensorflow).
 
-Note, this is a duct-tape POC. Using this in production will result in multiple RGEs.
+## What's new
+
+This is the initial release of the `tensorsets` repo.
+
+# Known issues
+
+This is a duct-tape POC. Using this in production will result in multiple RGEs.
```

Comment:
Elson, what is an RGE. Fred and Rene did not recognize it. Even Google didn't tell me. I would like to spell it out on first use along with the initials. Regards, Dan